README.md

```diff
@@ -134,9 +134,10 @@ Important options are
   Compiling with maximum rank 15 can be resource intensive and requires at least 16 GB of memory to allow parallel compilation or 4 GB memory for sequential compilation.
 - `-DBUILD_SHARED_LIBS` set to `on` in case you want link your application dynamically against the standard library (default: `off`).
 
-For example, to configure a build using the Ninja backend and generating procedures up to rank 7, which is installed to your home directory use
+For example, to configure a build using the Ninja backend while specifying compiler flags `FFLAGS`, generating procedures up to rank 7, and installing to your home directory, use
 
 ```sh
+export FFLAGS="-O3 -flto -fPIC"
```

Comment:
What is reason to request position independent code via the Fortran flags? This should be done via the [`CMAKE_POSITION_INDEPENDENT_CODE`](https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html#variable:CMAKE_POSITION_INDEPENDENT_CODE) variable instead if needed. Also for the manual makefile build this flag is absent.

Reply:
Thanks for the comment. I think `-fPIC` should be removed, and will implement that. 

The reason I added it here is because it was part of the performance solution discussed [here](https://github.com/fortran-lang/stdlib/issues/524), that @jvdp1 figured out. I wasn't aware of other means to specify it. 

However I just checked now, and find `-fPIC` it isn't needed to get good performance in that case -- just `-flto`. So best to avoid the extra complexity here.

Reply:
Position independent code is usually used to link shared object, for linking executables from static libraries it is not required. Building with `-DBUILD_SHARED_LIBS=true` to get shared objects will automatically enable position independent code. Enabling it for static libraries is usually only required if you plan to manually link it into a shared object later.

Reply:
I think the `-fPIC` option is important if you want to use stdlib as a shared library. But it should be discussed in a different section (where also @awvwgk's answer for CMake can be added).